requirements/requirements-django-py-3.5.txt

```diff
@@ -0,0 +1,151 @@
+aiida-export-migration-tests==0.8.0
```

Comment:
Should we leave these files in the visible `requirements/` directory or should we make them hidden?

Reply:
good question. let's leave them here for the moment, and then see how gio/seb about it.